website/docs/api/architectures.mdx

```diff
@@ -1043,6 +1043,9 @@ A neural network model where token vectors are calculated using a CNN. The
 vectors are mean pooled and used as features in a feed-forward network. This
 architecture is usually less accurate than the ensemble, but runs faster.
 
+This model is identical to [TexCatReduce.v1](#TextCatReduce) with
+`use_reduce_mean=true`.
```

Comment:
Should we add that the other two should be set to `False`?

Reply:
Done.